dev/tests/acceptance/tests/functional/Magento/FunctionalTest/Store/Page/AdminSystemWebsiteEditPage.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<pages xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../../vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Page/etc/PageObject.xsd">
+    <page name="AdminSystemStoreEditPage" url="system_store/editWebsite" module="Store" area="admin">
```

Comment:
module should be filled by this template `<VendorName>_<ModuleName>` (`Magento_Store`)